### ISSUE TITLE
Show archinstall version at the end of output if erroring

### ISSUE THREAD

**Issue by C0rn3j** [archlinux/archinstall#2423]
![image](https://github.com/archlinux/archinstall/assets/1641362/e7e06871-ef15-486c-9a53-b64f548d20c3)

This is the kind of picture/screenshot that is often found in support groups.   
It would help immensely if archinstall version was always printed in the error strings.

My suggestion would be marking such error messages with `[archinstall 2.10.2]` or a similar string to identify the version immediately.

`archinstall[2.10.2]: Would you like to re-try this download? (Y/n)`

I was also about to suggest a version check, but I see that @svartkanin already implemented that - thank you! https://github.com/archlinux/archinstall/pull/2169

**Comment by svartkanin** [comment]
When reporting errors and bugs, the expectation is that the installation log file is provided which will provide all the necessary information on the confirmation and current version

**Comment by C0rn3j** [comment]
> the expectation is that the installation log file is provided

I am well aware. Unfortunately, the practical reality is that people just post pictures like the example above with no additional text, and volunteers then waste their time on getting information out of them, only to figure out they're running some old version 5-10 minutes later after wasting time getting the person to post further logs and information, instead of just telling them to update.

I do think the merged PR is a great help in this regard, and adding the version to the output on the screen would just be a nice finishing touch.

---

Something else I just thought about, setting a temporary alias (`archinstalll-upload-log`?`) that would execute the suggested upload command would be a nice touch, so people don't have to type such a long command with lots of special characters, making it more likely they do so.  

Or asking them if they would like to upload their log to get a log URL directly.

**Comment by svartkanin** [comment]
Fair point, we can definitely do a version check again and show that they are running an older version.
I like the idea of providing a option to help uploading the log. Maybe when an errors appears we could have a prompt for uploading the the log file.

**Comment by cbrbygones** [comment]
I second this idea, and I also think a log upload flag would be a wonderful addition too. Where should it be uploaded to, though? Would it be on Arch's official repos/servers? Sorry if I'm chiming in late.

**Comment by C0rn3j** [comment]
> Fair point, we can definitely do a version check again and show that they are running an older version.

Version checking again would be even nicer, but my idea is to show the version even if it IS up to date, so it is obvious from a picture like this that the person is running an old version (simply by knowing what the latest is).

One could use [0x0](https://git.0x0.st/mia/0x0) or a self hosted instance of it, question is who would be hosting it, so for now it could go in against the public instance?

```bash
<command to print output> |& curl -F 'file=@-' https://0x0.st
```

**Comment by svartkanin** [comment]
Under the error we're already displaying a hint of how and where to upload the log file. Is that not sufficient?

**Comment by C0rn3j** [comment]
Would you have a screenshot of how the current state looks like?
